packages/cloudflare/src/cli/build/index.ts

```diff
@@ -94,6 +97,34 @@ export async function build(projectOpts: ProjectOptions): Promise<void> {
   logger.info("OpenNext build complete.");
 }
 
+/**
+ * Creates a `open-next.config.ts` file for the user if it doesn't exist, but only after getting the user's confirmation.
+ *
+ * If the users refuses an error is thrown (since the file is mandatory).
+ *
+ * @param baseDir the Next.js app root folder
+ */
+async function createOpenNextConfigIfNotExistent(baseDir: string): Promise<void> {
+  const openNextConfigPath = join(baseDir, "open-next.config.ts");
+
+  if (!existsSync(openNextConfigPath)) {
+    const questionName = "create-open-next-config";
+    const answer = (
+      (await Enquirer.prompt({
+        name: "create-open-next-config",
+        message: "Missing required `open-next.config.ts` file, do you want to create one?",
+        type: "confirm",
+        initial: "y",
+      })) as { [questionName]: boolean }
+    )[questionName];
+    if (!answer) {
+      throw new Error("A `open-next.config.ts` file is mandatory, aborting!");
+    }
+
+    cpSync(`${import.meta.dirname}/templates/defaults/open-next.config.ts`, openNextConfigPath);
```

Comment:
That will not work any more after my pending PR compiling all the files.

Maybe return that as a string instead?

Reply:
what do you mean by "return as a string"? inlining the file's content as a string here?

Reply:
That's one possibility.

Or you can copy the compiled .js and rename it to .ts (but probably we want some types later)

Or you can create a /template folder alongside /src, add it to the npm package and use that

...